Makefile

```diff
@@ -185,19 +185,19 @@ gotidy:
 update-dep:
 	$(MAKE) for-all-mod CMD="$(PWD)/internal/buildscripts/update-dep"
 
-STABLE_OTEL_VERSION=v1.13.0
-UNSTABLE_OTEL_VERSION=v0.36.0
-UNSTABLE_CONTRIB_OTEL_VERSION=v0.40.0
-STABLE_CONTRIB_OTEL_VERSION=v1.15.0
-UNSTABLE_COLLECTOR_VERSION=v0.72.0
-STABLE_COLLECTOR_VERSION=v1.0.0-rc6
-UNSTABLE_COLLECTOR_CONTRIB_VERSION=v0.72.0
+  STABLE_OTEL_VERSION=v1.15.0-rc.1
+UNSTABLE_OTEL_VERSION=v0.38.0-rc.1
+  STABLE_CONTRIB_OTEL_VERSION=v1.16.0-rc.1
+UNSTABLE_CONTRIB_OTEL_VERSION=v0.41.0-rc.1
+  STABLE_COLLECTOR_VERSION=v1.0.0-rc7
```

Comment:
instead of bumping to `rc` tags now, would it be worth just waiting for the GA tags of each of these?

Reply:
@damemi follow your opinion. However, I will continue to send PRs to follow upstream.